imagetagger/imagetagger/wsgi.py

```diff
@@ -9,8 +9,9 @@
 
 import os
 
-from django.core.wsgi import get_wsgi_application
+from configurations.wsgi import get_wsgi_application
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "imagetagger.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "Prod")
```

Comment:
I think this should be `DJANGO_CONFIGURATION`

Reply:
of course. it is fixed in ff8ed69be091cfc3c6d67e7ebe93538110602238